backend/src/routes/middleware/isAuthorized.ts

```diff
@@ -42,26 +42,26 @@ export function checkIfUserAuthorized(
             return _buildErrorResponse(response, 'TOKEN NOT PROVIDED');
         }
 
-        const token = await VerifyAndDecodeJWT(tokenString);
+        const token = await verifyAndDecodeJWT(tokenString);
 
         if (token instanceof Error) {
             return _buildErrorResponse(response, 'INVALID TOKEN');
         }
 
-        const user = await GetUserById(token.userId);
+        const user = await new UserResolver().getUser(token.userId);
```

Comment:
This is kinda terrible, but there's no dependency injection framework on the backend yet